docusaurus/docs/iOS/swiftui/chat-channel-components/overview.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Overview
+---
+
+## What is the Chat Channel?
+
+![Depiction of different message components in the example of the message list.](../../assets/message-components-overview.png)
+
+The chat channel describes all the content that is shown for a specific channel. It consists of three main parts:
+
+- Channel Header
+- Message List
+- Message Composer
+
+There are **many components** that make of the chat channel screen. The **channel header** is shown on the top to give an overview over the channel as well as provide other **helpful information**. It can take care of navigation (e.g. using a back button) and allow the user to see more information about members, name, and other properties of a channel. All of these components are **customizable and easily replaceable**.
+
+The **message list** in the SwiftUI SDK is the view that shows all the messages in a channel. These include all kinds of types like **text, images, links, GIFs, and many more**. Also, it includes features such as message **reactions** and **threads**.
+
+In order to compose messages there is the message **composer**. It can not only send messages but also all **other kinds of attachments**, such as photos, videos, voicemails, and many more. Also, it is possible to build **customized, personalized attachments** tailored for specific use-cases. The composer also supports advanced features such as **message commands** using e.g. the `/` shortcut.
+
+![An image discriminating the different parts of the chat channel.](../../assets/chat-channel-components.png)
+
+## Customizing Message Components
+
+The modular architecture of the Stream Chat SwiftUI SDK allows developers to change and customize the message components to match their functionalities and needs.
+
+There are dedicated sections for each of the components that can be customized with detailed explanations how to do it:
+
+- [Channel Header](../channel-header)
+- [Message List](../message-list)
+- [Message Composer](../message-composer)
+
+In case there is anything missing from that list that you'd like to see, then please [let us know](https://twitter.com/getstream_io).
+
+:::info
+There is more content to be found on our [Youtube channel](https://www.youtube.com/channel/UC2xOn0xQj1HIpHJpOy5tvpA) in a video about [recreating the message list from iMessage](https://youtu.be/8Nkmk85H8HQ).
+:::
+
+## Showing a Channel programmatically
+
+While the general use-case is that users navigate from a Channel List to a chat channel there can be exceptions from that. In some cases, it is required to directly show a channel. Not only that, it can also be helpful to have the option to programmatically navigate to a certain channel.
```

Comment:
```suggestion
While the general use case is that users navigate from a Channel List to a chat channel, there can be exceptions to that. In some cases, it is required to directly show a channel. Not only that, but it can also be helpful to have the option to programmatically navigate to a certain channel.
```